security/advancedtls/advancedtls.go

```diff
@@ -234,9 +234,16 @@ func (o *ServerOptions) config() (*tls.Config, error) {
 		clientAuth = tls.RequireAnyClientCert
 	}
 	config := &tls.Config{
-		ClientAuth:     clientAuth,
-		Certificates:   o.Certificates,
-		GetCertificate: o.GetCertificate,
+		ClientAuth:   clientAuth,
+		Certificates: o.Certificates,
+	}
+	// The function getCertificateWithSNI is only able to perform SNI logic for go1.10 and above.
+	// It will return the first certificate in o.GetCertificate for go1.9.
```

Comment:
This comment would be better served if placed at the `GetCertificate` function comment. Users should know what to expect by reading the godoc.

Reply:
Sure!